src/libs/ReportUtils.js

```diff
@@ -873,7 +873,7 @@ function hasReportNameError(report) {
  */
 function getParsedComment(text) {
```

Comment:
Can you please move the comments from `buildOptimisticAddCommentReportAction` to `getParsedComment` looks they were left there by mistake. Also please mention that even though we skip parsing for long texts we still escape the text. 

Comments:
```
    // For comments shorter than 10k chars, convert the comment from MD into HTML because that's how it is stored in the database
    // For longer comments, skip parsing and display plaintext for performance reasons. It takes over 40s to parse a 100k long string!!
```

Reply:
Makes sense. Will update shortly!